app/views/tag/show/_sort.html.erb

```diff
@@ -15,6 +15,9 @@
     <a class="dropdown-item <% if params[:order] == 'likes' %> active<% end %>" href="?order=likes">
       <i class="fa fa-star"></i> <span class="d-lg-inline">by likes</span>
     </a>
+    <div class="dropdown-divider"></div>
```

Comment:
Hi! I wonder if it'd be just a little clearer with this extra header. But otherwise this looks tremendous! Thank you so much!

```suggestion
    <div class="dropdown-divider"></div>
    <h6 class="dropdown-header">View date range</h6>
```